### ISSUE TITLE
[Rule] misplaced not in condition 

### ISSUE THREAD

**Issue by bhirsz** [MarketSquare/robotframework-robocop#893]
Temporary naming for the issue. It should detect:

```
IF not ${var} is None
WHILE not ${var} in @{list}
```

**Comment by mnojek** [comment]
I don't fully get it. What's the issue here and what should be the recommended code?

**Comment by bhirsz** [comment]
It should be:

```
IF ${var} is not None
WHILE ${var} not in @{list}
```
